Tests/SKCoreTests/CompilationDatabaseTests.swift

```diff
@@ -187,6 +187,25 @@ final class CompilationDatabaseTests: XCTestCase {
     XCTAssertNotNil(tryLoadCompilationDatabase(directory: AbsolutePath("/a"), fs))
   }
 
+  func testFixedCompilationDatabase() {
```

Comment:
Please regenerate Linux tests

```
swift build --regenerate-linuxmain
```

If your version of swiftpm is too new and no longer supports that you can add the new test manually to `Tests/SKCoreTests/XCTestManifests.swift`